### DISCUSSION TITLE
Importing CSV with links [Oct 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6678] [Discord Threads]
> 🙋‍♂️ **craige3666**  
## I'm looking to import a csv with one of the fields as a record in another table of the database - but's not importing the link to table variable. (Name and cases match). It's a cities list wit their respective countries attached. The countries field links to the list of respective countries in the countries table. Any workaround or is it just my version of nocodedb that's the issue. How do I get my list of 150+ cities imported into the cities table and mapped to their respective countries (in the countries table) without having to link them one by one . The row headers are different in my instant and are mapped correctly in there.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1148254063062941779/image.png?ex=65367a6e&is=6524056e&hm=30c4fc76d1667d466992c43fccb355d6395f6e7c7135ce463224eea89401a829&)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@882078611035815957> This is not possible in UI. You will have to write a script using APIs to get it done quickly.    
####    
#### You can create two tables City & Country to import initial unique data set. Then either link manually or use a script / APIs. If you are linking multiple records- use `ALT + Link` when linking records. It makes it slightly easier allowing multiple records to be linked in one go, but yes- its manual work still  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1148254063461412864)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
